fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBInQueryTest.java

```diff
@@ -150,7 +150,7 @@ void testInQueryNoIndex() throws Exception {
                             .where(predicates(valuePredicate(fieldValue("num_value_2"), new Comparisons.ListComparison(Comparisons.Type.IN, ImmutableList.of(0, 2))))));
 
             assertEquals(997592219, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(-951957746, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
+            assertEquals(-1123732652, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
```

Comment:
why do we assert `Hashcode` in unit tests? How did we get the expected the `Hashcode`?

Reply:
This is the most compact way to guard against _any_ change to the structure of plans. We have discussed ways of making it more robust at some point. If the plan has legitimately changed, or if the `planHash` has legitimately changed, then the test failure itself has the new hash code. But care is required to understand the change before any such update.